### DISCUSSION TITLE
ARGOMENTO-ATTIVITÀ

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#134] [Idee e proposte]
Buongiorno,sarebbe possibile dividere la sezione argomento e attività in due aree distinte?Lo chiedo perché vedo che anche gli altri registri sono divisi in orario,materia,docente,argomento,compiti etc... in modo da avere più ordine e capire subito quale argomento di preciso si è stato trattato e quali compiti inerenti a quella materia sono stati assegnati.

**Comment by trinko** [comment]
Parli della visualizzazione per i docenti, immagino, perché per alunni/genitori è già separata.
La segno tra le richieste, ma modifiche all'interfaccia grafica non sono previste per quest'anno.

**Comment by sempliciniluigi89** [comment]
Certo certo,allego foto di quello che intendo
https://user-images.githubusercontent.com/93823125/150651634-6f169eff-8f63-4b12-9450-692efe2aa7bb.png

**Comment by sempliciniluigi89** [comment]
Certo certo

Il Sab 22 Gen 2022, 19:47 Antonello Dessì ***@***.***> ha
scritto:
